java/samples/sample-code/src/main/java/com/microsoft/semantickernel/Example99_BlogAnnouncement.java

```diff
@@ -0,0 +1,70 @@
+package com.microsoft.semantickernel;
+
+import com.azure.ai.openai.OpenAIAsyncClient;
+import com.azure.ai.openai.OpenAIClientBuilder;
+import com.azure.ai.openai.models.NonAzureOpenAIKeyCredential;
+import com.microsoft.semantickernel.builders.SKBuilders;
+import com.microsoft.semantickernel.connectors.ai.openai.util.AIProviderSettings;
+import com.microsoft.semantickernel.connectors.ai.openai.util.OpenAISettings;
+import com.microsoft.semantickernel.planner.actionplanner.Plan;
+import com.microsoft.semantickernel.planner.sequentialplanner.SequentialPlanner;
+import com.microsoft.semantickernel.skilldefinition.annotations.DefineSKFunction;
+import com.microsoft.semantickernel.skilldefinition.annotations.SKFunctionInputAttribute;
+import com.microsoft.semantickernel.skilldefinition.annotations.SKFunctionParameters;
+import com.microsoft.semantickernel.textcompletion.TextCompletion;
+
+public class Example99_BlogAnnouncement {
+
+  static {
+    System.setProperty("client.openai.key", "");
+  }
+
+  public static void main(String[] args) {
+    OpenAISettings settings = AIProviderSettings.getOpenAISettingsFromSystemProperties();
+
+    NonAzureOpenAIKeyCredential credential = new NonAzureOpenAIKeyCredential(settings.getKey());
+
+    OpenAIAsyncClient client = new OpenAIClientBuilder()
+        .credential(credential)
+        .buildAsyncClient();
+
+    TextCompletion textCompletionService = SKBuilders.textCompletionService().build(client, "text-davinci-003");
```

Comment:
`text-davinci-003` is no longer deployable in Azure. It would be better to switch to `gpt-35-turbo`. Once Johns PR to allow a chat completion service to be used for text completion this will be a straight forward change.